hw/dv/sv/mem_bkdr_if/mem_bkdr_if.core

```diff
@@ -10,6 +10,9 @@ filesets:
     depend:
       - lowrisc:opentitan:bus_params_pkg
       - lowrisc:dv:dv_utils
+      - lowrisc:dv:crypto_dpi_prince:0.1
+      - lowrisc:prim:all
```

Comment:
Could you create a core file for `prim_cipher_pkg` and just add dependency on that? I dont think we need to depend on all of it.

Reply:
yup I agree that is a better idea. Done.